reroute/src/Web/Routing/Router.hs

```diff
@@ -20,10 +21,10 @@ import qualified Data.Text as T
 
 newtype RegistryT n b middleware reqTypes (m :: * -> *) a
     = RegistryT
-    { runRegistryT :: RWST (Path '[]) [middleware] (RegistryState n b reqTypes) m a
+    { runRegistryT :: RWST (SafePath '[] 'Open) [middleware] (RegistryState n b reqTypes) m a
```

Comment:
Why does the registry only handle "open" `Path`s? I think we should remove the open parameter as soon as we leave the user facing part as it's really only there to make sure that a user writes useful `Path` patterns.